src/main/java/org/togetherjava/formatter/tokenizer/Token.java

```diff
@@ -0,0 +1,13 @@
+package org.togetherjava.formatter.tokenizer;
+
+import java.util.List;
+
+public record Token(String content, TokenType type) {
+    private static final List<TokenType> displayTypes =
+            List.of(TokenType.IDENTIFIER, TokenType.UNKNOWN, TokenType.STRING);
+
+    @Override
+    public String toString() {
+        return type().name() + (displayTypes.contains(type()) ? "(" + content() + ")" : "");
```

Comment:
I personally am not a big fan of ternary operators, but that might be me nitpicking, they often make it harder to read than necessary.